paddle/fluid/operators/mkldnn/lrn_mkldnn_op.cc

```diff
@@ -63,7 +63,8 @@ class LRNMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
         dims, platform::MKLDNNGetDataType<T>(), x->format());
 
     const std::string key = platform::LRNMKLDNNHandler::GetHash(
-        dims, n, alpha, beta, k, x->format(), ctx.op().Output("Out"));
+        dims, n, alpha, beta, k, x->format(),
+        std::to_string(*reinterpret_cast<int*> out));
```

Comment:
Added code is basically extracting first four bytes(integer) of Tensor object . First four bytes in Tensor for MKL-DNN is:
https://github.com/PaddlePaddle/Paddle/blob/561232c25a0af9fb408f086faf6f443fa0140d1c/paddle/fluid/framework/tensor.h#L57

Which describe what mkl-dnn format is used for data arrangement in given Tensor. In most cases it will be value of 40 (nchw8c) or 41 (nchw16c). Using this value does not help as it will be almost always value of 40 or 41. 

I understand that this change is more like to show some idea

Reply:
> I understand that this change is more like to show some idea

Yes, it is only to show that we will remove the `Input` and `Output` interface later.